### PR TITLE
[iOS] Migration to Google Mobile Ads SDK 11

### PR DESCRIPTION
**Pull request by Astrovic**
- Migration to SDK v11 (https://developers.google.com/admob/ios/migration#migrate-to-sdk-v11)
- Update Google Mobile Ads SDK 11.0.1 (https://github.com/CocoaPods/Specs/tree/master/Specs/5/9/a/Google-Mobile-Ads-SDK/11.0.1)
- Update InMobi SDK 10.6.4 (https://support.inmobi.com/monetize/sdk-documentation/download-sdk)
- Update InMobiAdapter 10.6.0.0 (https://developers.google.com/admob/ios/mediation/inmobi#version-10.6.0.0)
- Update Meta Audience Network SDK [6.14.0](https://developers.facebook.com/docs/audience-network/setting-up/platform-setup/ios/changelog#6_14_0)
- Update Meta Adapter [6.14.0.0](https://developers.google.com/admob/ios/mediation/meta#version-6.14.0.0)
- Added iphone property: `tagForChildDirectedTreatment`. https://developers.google.com/admob/ios/targeting#child-directed_setting for more infos
- Added iphone property: `tagForUnderAgeOfConsent`.https://developers.google.com/admob/ios/targeting#users_under_the_age_of_consent for more infos
- Added iphone property: `maxAdContentRating`. https://developers.google.com/admob/ios/targeting#ad_content_filtering for more infos
- Deprecated iOS `Admob.SIMULATOR_ID`
- Added iphone methods `isGDPR()`, `canShowAds()`, `canShowPersonalizedAds()` (thanks to [deckameron](https://github.com/deckameron))
- Updated documentation and iOS example

Module file: [ti.admob-iphone-7.0.0.zip](https://github.com/Astrovic/ti.admob/releases/download/ios-7.0.0/ti.admob-iphone-7.0.0.zip)

⚠️ **PLEASE NOTE**
Currently, Facebook has not yet released an SDK compatible with the new Google Mobile Ads SDK 11. So if you use Facebook mediations, use the previous AdMob module version [6.3.0](https://github.com/Astrovic/ti.admob/releases/tag/ios-6.3.0)

### CODE REVIEW

File: apidoc/Admob.yml [hunk 1]
```diff
@@ -444,6 +447,27 @@ methods:
         type: Callback<LoadFormObject>
         summary: async callback called when retrieved
 
+  - name: isGDPR
+    returns:
+      type: Boolean
+    summary: Check in the IABTCF string if GDPR applies, so if in EEA.
+    platforms: [iphone, ipad]
+    since: "2.6.3"
```
Comment (hansemannn) [inline apidoc/Admob.yml:455]: Aren't these added in this version as well?

File: ios/documentation/index.md [hunk 2]
```diff
@@ -185,6 +196,29 @@ To check if a form is available, use the callback status parameter
...
+
+Finally you know if you can request AdMob Personalized or Non-Personalized Ads, if Non-Personalized you have to forward the request using this snippet.
+
+```js
+  var Admob = require('ti.admob');
+
+  var ad = Admob.createView({
+    // your properties...
+    extras: { 'npa': "1"}, // npa=1 disables personalized ads 
+  });
```
Comment (hansemannn) [inline ios/documentation/index.md:219]:
```suggestion
import Admob from 'ti.admob';

const ad = Admob.createView({
    // your properties...
    extras: { 'npa': '1' }, // npa=1 disables personalized ads 
});
```

**Review by hansemannn — CHANGES_REQUESTED** [review]
Just minor improvements, the rest looks awesome, especially the well-documented new APIs. Thank you!

File: apidoc/Admob.yml (hunk 1 above)
Comment (Astrovic) [reply apidoc/Admob.yml]: No, they were introduced in 6.3.0, it was a typo. I just corrected it.

**Comment by Astrovic** [comment]
Well, I applied the changes!
In the first comment I also added a link to the zip file of the module and a note relating to Facebook mediation.

**Review by hansemannn — APPROVED** [review]

**Comment by hansemannn** [comment]
Thank you! Is the attached zip in the PR the latest one?

**Comment by Astrovic** [comment]
Yes it is the latest version

**Comment by hansemannn** [comment]
It's released!

**Pull request merged**